common/djangoapps/course_modes/tests/test_signals.py

```diff
@@ -87,3 +89,28 @@ def test_verified_mode_explicitly_set(self, verification_window):
             course_mode.refresh_from_db()
 
             self.assertEqual(course_mode.expiration_datetime, self.end - timedelta(days=verification_window))
+
+    def test_masters_mode(self):
+        # create an xblock with verified group access
+        verified_section = ItemFactory.create(
+            category="sequential",
+            metadata={'group_access': {50: [2]}}
+        )
+        # and a section with no restriction
+        section2 = ItemFactory.create(
+            category="sequential",
+        )
+        section3 = ItemFactory.create(
+            category='sequential',
+            metadata={'group_access': {50: [1]}}
+        )
+        with self.store.branch_setting(ModuleStoreEnum.Branch.draft_preferred):
```

Comment:
What does this do?

Reply:
I don't really know, but the tests were failing without it...